en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,54 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+In order to have minimum constraints on contents, Open Orchestra allows you to store any type
+of ``value`` in the ``ContentAttribute``.
+Sometimes, for instance in data tables, there is no easy way to render the value to the backoffice user so we have added
```

Comment:
I would rather suppress the sometimes and say `When it comes to display the value, in a datatable for instance,...`